src/app/room/[slug]/page.module.css

```
@@ -0,0 +1,187 @@
+}
+
+.container {
+  --panel-horizontal-padding: 24px;
+  --panel-content-width: 640px;
+  --control-bar-width: min(372px, calc(100% - 8px), calc(100vw - 48px));
+  --control-bar-height: calc(var(--control-bar-width) * 76 / 372);
+  --control-bar-bottom-offset: 24px;
+  --control-bar-reserved-space: calc(
+    var(--control-bar-height) + var(--control-bar-bottom-offset) + 16px
+  );
+  position: relative;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)